src/events/components/DetailView/PublicAttendees/index.tsx

```diff
@@ -1,29 +1,46 @@
-import React, { FC, useEffect } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 
 import { useDispatch, useSelector } from 'core/redux/hooks';
 import { selectPublicAttendeesForEventId } from 'events/selectors/publicAttendee';
 import { fetchPublicAttendeesByEventId } from 'events/slices/publicAttendees';
 
 import { Attendee } from './Attendee';
 import style from './PublicAttendees.less';
+import { IPublicAttendee } from 'events/models/Attendee';
+import { Button, Modal } from '@dotkomonline/design-system';
 
 interface IProps {
   eventId: number;
+  isAttending: boolean;
 }
 
-export const PublicAttendees: FC<IProps> = ({ eventId }) => {
+const attendeeList = (attendees: IPublicAttendee[]) => (
+  <div className={style.grid}>
+    {attendees.map((attendee, index) => (
+      <Attendee key={attendee.id} attendee={attendee} count={index + 1} />
+    ))}
+  </div>
+);
```

Comment:
Why is this a function and not a component?

Reply:
I started it of as a function component, but then switched back to a function. It is just personal preference, but I did it because:

- When only passing 1 variable, I don't think it matters much (visually) to write `{attendeeList(attendees)}` compared to <AttendeeList attendees={attendees} />
- When using only 1 variable, I feel it is redundant to define the props. Writing an interface creates 3 more lines of text and a couple of lines of white space
- I could also define the props inline, but that makes it harder to read what is happening at first glance, which I am not that much of a fan of.

I can change it back, if it is wanted, but it was just for my personal (visual noise) reasons.